src/renderer/renderScene.ts

```diff
@@ -22,9 +22,13 @@ type HandlerRectanglesRet = keyof ReturnType<typeof handlerRectangles>;
 function colorsForClientId(clientId: string) {
   // Naive way of getting an integer out of the clientId
   const sum = clientId.split("").reduce((a, str) => a + str.charCodeAt(0), 0);
+
+  // Skip transparent background.
+  const backgrounds = colors.elementBackground.splice(1);
+  const strokes = colors.elementStroke.splice(1);
```

Comment:
Shouldn’t this be `slice`? `splice` is the one that mutates.

Reply:
oups...